documentation/docs/tutorials/validate_university_degree.mdx

```diff
@@ -282,13 +283,13 @@ removeMerkleKey(
     KeyCollection.fromJSON(issuer.verifKey));
 ```
 
-Note that you could also revoke Alice's signature on the verifiable presentation, by removing her verification method.
+You could also revoke Alice's signature on the verifiable presentation by removing her verification method.
 
 ### 11. **Verifier**: Verify Signatures Again
 
-The IOTA Foundation verifies Alice's and the University's signatures again by checking the verifiable presentation and finds out that the University revoked their signature.
+The IOTA Foundation verifies Alice and the University's signatures again by checking the verifiable presentation and discovers that the University revoked her signature.
```

Comment:
```suggestion
The IOTA Foundation verifies Alice and the University's signatures again by checking the verifiable presentation and discovers that the University revoked their signature.
```

I think it's the university that revoked the signature after they find out Alice cheated on line 249

Reply:
Then it should be "its signature" :P

Reply:
👌